.github/workflows/publish-canary.yml

```diff
@@ -53,9 +53,12 @@ jobs:
 
       # Release
       - name: Prerelease Version and Publish
+        id: changesets
         run: |
           yarn config set -H 'npmAuthToken' "${{secrets.NPM_TOKEN}}"
-          git config --global user.email "engineering@propeldata.com"
-          git config --global user.name "Propel Data Cloud, Inc." 
-          yarn changeset version --snapshot
-          yarn release --pre
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > .npmrc
+          yarn changeset version --snapshot canary
+          yarn changeset publish --tag canary --no-git-tag
```

Comment:
I used the same commands in the [POC project](https://github.com/sashathor/monorepo-lib/blob/main/.github/workflows/publish-snapshot.yml#L33) and even [recorded a demo](https://propeldatacloud.slack.com/archives/C01T1MZG6HE/p1703255345509439) based on it. Now I'm wondering why I didn't include it in [the actual PR](https://github.com/propeldata/ui-kit/commit/313d6a571d4ad9a4dfd42e97f7c36baf4dc534a2) 🤔.
The current commands also work sometimes, but it some point I also got the same message `warn No unreleased changesets found, exiting.` So, let's proceed with the fix.
<img width="758" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/f910caae-3f41-4dfe-ba8a-6abb2852cdf1">


Anyway, thank you for the fix. Could you please separate the canary fix from `chartConfigProps`?

Reply:
I set the fix in this PR: https://github.com/propeldata/ui-kit/pull/184